runtime/calamari/src/lib.rs

```diff
@@ -491,7 +493,9 @@ construct_runtime!(
 	{
 		// System support stuff.
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
-		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Storage, Inherent, Event<T>} = 1,
+		ParachainSystem: cumulus_pallet_parachain_system::{
+			Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned,
```

Comment:
What's the ``ValidateUnsigned`` for actually ?

Reply:
In general, substrate allows unsigned transaction, which means there's no one to sign this transaction.
I suppose there's one unsigned extrinsic in `ParachainSystem`.

BTW, if we want to implement `ValidateUnsigned` for our own pallet, we should be careful, because that would be point someone might attack us.